deployments/apiv2/services/users/src/main.py

```diff
@@ -776,7 +776,7 @@ async def update_user(
                 )
         else:
             if action in ("deactivate"):
```

Comment:
I meant to make this `if action == "deactivate":`, I think got mixed up when fixing merge conflicts